### PR TITLE
Add Numi.app latest version

### PR DESCRIPTION
**Pull request by vinkla**
Calculator app for Mac you will love to use. http://numi.io/

### CODE REVIEW

File: Casks/numi.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask :v1 => 'numi' do
+  version :latest
+  sha256 :no_check
+
+  # devmate.com is the official download host per the vendor homepage
+  url 'http://dl.devmate.com/com.dmitrynikolaev.numi/beta/Numi.zip'
+  name 'Numi'
+  homepage 'http://numi.io/'
+  license :gratis
```
Comment (vitorgalvao) [inline Casks/numi.rb]: Are you sure it will remain so, after the beta?

Comment (vinkla) [inline Casks/numi.rb]: Maybe not. How should we do it instead?

Comment (vitorgalvao) [inline Casks/numi.rb]: Asking the developer is the best way; that’s what I always do. Twitter seems to be the fastest way.

Comment (vinkla) [inline Casks/numi.rb]: Okay, I've sent them an email. Lets wait for them to respond.

Comment (vinkla) [inline Casks/numi.rb]: @vitorgalvao Fixed!

**Pull request merged**